pallets/sidechain/src/tests.rs

```diff
@@ -208,6 +208,7 @@ fn register_enclave(signer_pub_key: &MrSigner, cert: &[u8], expected_enclave_cou
 		cert.to_vec(),
 		URL.to_vec(),
 		None,
+		None,
```

Comment:
this is a fix for another PR?

Reply:
It was fix in https://github.com/litentry/litentry-parachain/pull/1369, I will revert this.